app/frontend/src/constants.ts

```diff
@@ -7,4 +7,15 @@ export const userLocationDefaultRadius = 500;
 
 export const pingInterval = 10000;
 
-export const reactQueryRetries = 3;
+export const reactQueryRetries = 2;
+
+export const grpcTimeout = 10000; //milliseconds
+
+export const grpcErrorStrings = {
+  "Deadline exceeded":
+    "Server took too long to respond. Please check your Internet connection or try again later.",
+  "Http response at 400 or 500 level":
+    "Couldn't find the server. Please check your Internet connection or try again later.",
```

Comment:
"find" is a confusing term for end users, I think. "Couldn't connect to server"?